rs/sns/governance/src/governance.rs

```diff
@@ -4632,6 +4636,84 @@ impl Governance {
         self.maybe_move_staked_maturity();
 
         self.maybe_gc();
+
+        {
```

Comment:
We've typically been extracting requests that run every heartbeat into a "maybe_do_x" pattern.  If this should not run on every heartbeat, but only when other things aren't happening, then separating out the "should_do_X" from the "do_X" makes sense.

Reply:
So this LGT you? :-)